src/cookbook/forms/focus.md

```diff
@@ -49,8 +49,10 @@ First, we'll need to create a [`FocusNode`](https://docs.flutter.io/flutter/widg
 We will use the `FocusNode` to identify a specific `TextField` in Flutter's
 "focus tree." This will allow us to focus the `TextField` in the next steps.
 
-Since focus nodes are long-lived objects, we need to store them in a `State`
-class. In addition, we need to `dispose` of them when they're no longer needed!
+Since focus nodes are long-lived objects, we need to manage the lifecycle 
```

Comment:
Generally, "you" should be used instead of "we" (but I realize that this means revisiting the entire recipe, which should probably be addressed in a separate PR). /cc @Sfshaza

Reply:
Good feedback -- unfortunately I wasn't aware of that and have been using "we" for all recipes! Oh no :/

Lemme know if you want me to prepare a separate PR which changes that across the recipes.

Reply:
No worries, the switch in guideline is relatively recent (or at least our systematic application of it across the three Dart sites). I'll let @Sfshaza chime in; I don't know what else you are juggling, so I can't comment on relative priorities.

Reply:
Yes, Developer Guidelines prefer "you" over "we". https://developers.google.com/style/person

This rule is not that recent, but Matt approved the original PRs and doesn't know the guidelines. So I didn't mention it. But maybe just change current guidelines to "you" and we can worry about it later.